framework/include/problems/FEProblemBase.h

```diff
@@ -127,6 +127,18 @@ enum MooseLinearConvergenceReason
   MOOSE_DIVERGED_PCSETUP_FAILED = -11
 };
 
+enum MoosePicardConvergenceReason
+{
+  MOOSE_PICARD_ITERATING = 0,
+  MOOSE_PICARD_CONVERGED_NONLINEAR = 1,
+  MOOSE_PICARD_CONVERGED_ABS = 2,
+  MOOSE_PICARD_CONVERGED_RELATIVE = 3,
+  MOOSE_PICARD_CONVERGED_CUSTOM = 4,
+  MOOSE_PICARD_DIVERGED_MAX_ITS = -1,
+  MOOSE_PICARD_DIVERGED_NONLINEAR = -2,
+  MOOSE_PICARD_DIVERGED_FAILED_MULTIAPP = -3
```

Comment:
Moving it to MooseType might be better?

Reply:
We have `MooseNonlinearConvergenceReason` and `MooseLinearConvergenceReason` sitting right above this. Maybe move all of them?

Reply:
I guess it depends on how wide of scope we need. If we really don't need these status flags outside of the problem/executioner, they are fine here. Putting everything in the types file for the sake of having them all in one place may not be the best approach. I would say that this is fine if the other statuses are also currently sitting here.

Reply:
Will do.